src/ServicePulse.Host/vue/src/components/TabContainer.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+import { computed, inject, onBeforeMount, watch } from "vue";
+
+const addTab = inject("addTab");
+const activeTab = inject("activeTab");
```

Comment:
I don't like the concept of provide/inject for first level relations, but if we're going to then the injection key should be an exported symbol so that we can't have accidental key clashes